packages/api/portfolioDrafts/commonPortfolioDraftMockData.ts

```diff
@@ -0,0 +1,196 @@
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { mockApplicationStep } from "./application/commonApplicationMockData";
+import { mockFundingStep } from "./funding/commonFundingMockData";
+import { mockPortfolioStep } from "./portfolio/commonPortfolioMockData";
+import { PortfolioDraft } from "../models/PortfolioDraft";
+import { PortfolioDraftSummary } from "../models/PortfolioDraftSummary";
+import { ProvisioningStatus } from "../models/ProvisioningStatus";
+import { v4 as uuidv4 } from "uuid";
+
+const now = new Date().toISOString();
+
+export const mockPortfolioDraftSummary: PortfolioDraftSummary = {
+  id: uuidv4(),
+  status: ProvisioningStatus.NOT_STARTED,
+  updated_at: now,
+  created_at: now,
+  name: "",
+  description: "",
+  num_portfolio_managers: 0,
+  num_task_orders: 0,
+  num_applications: 0,
+  num_environments: 0,
+};
+
+export const mockPortfolioDraft: PortfolioDraft = {
+  ...mockPortfolioDraftSummary,
+  portfolio_step: mockPortfolioStep,
+  funding_step: mockFundingStep,
+  application_step: mockApplicationStep,
+  num_portfolio_managers: mockPortfolioStep.portfolio_managers.length,
+  num_task_orders: mockFundingStep.task_orders.length,
+  num_applications: mockApplicationStep.applications.length,
+  num_environments: mockApplicationStep.applications.flatMap((app) => app.environments).length,
+  name: mockPortfolioStep.name,
+};
+
+export const validRequest: APIGatewayProxyEvent = {
+  pathParameters: { portfolioDraftId: uuidv4() },
+} as any;
+
+/** ABOVE THIS LINE ARE VALID OBJECTS WITH GOOD DATA **/
+/** BELOW THIS LINE ARE INVALID OBJECTS WITH MISSING FIELDS AND BAD DATA **/
```

Comment:
```suggestion
/* ABOVE THIS LINE ARE VALID OBJECTS WITH GOOD DATA */
/* BELOW THIS LINE ARE INVALID OBJECTS WITH MISSING FIELDS AND BAD DATA */
```

just to prevent any chance of these being interpreted as JSDoc/TSDoc. Love the comment explaining. We may want to (eventually) consider whether there's a way we could put these in some sort of named data structure to clarify whether they're valid/invalid rather than relying on comments. but i see that this format is used pretty consistently so that's good; and this is definitely an improvement from where we were.

Reply:
Fixed in c58f0e988f9ce67bf1d68d860bbc01c9c1aeb6f9